runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -32,13 +28,6 @@
     LAYOUT_NOBOOTSTRAP(19, TypeToken.of(CorfuMsg.class), true, false),
 
     ERROR_SERVER_EXCEPTION(200, new TypeToken<CorfuPayloadMsg<ExceptionMsg>>() {}, true, false),
```

Comment:
I've created a new PR that removed legacy code. I decided against doing it in this PR since it is a substantial change (2K LoC). Once this PR goes in, I will rebase it. Take a look at the last commit: https://github.com/CorfuDB/CorfuDB/pull/2869/commits